commands/Moderation/mute.js

```diff
@@ -0,0 +1,59 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRoleId, reportChannelId, titleURL, muteRoleName } = msg.guild.settings;
+  try {
```

Comment:
Why do you wrap everything in a try/catch? That slows the command a lot, any rejected promise will be caught by the command handler.

Reply:
try/catch allows me to send the errors to a specific channel instead of having to read the logs. Is there an easier way to do this? When im not on computer i can see the errors appear on my #error-log and easily know what went wrong

Reply:
The errors you'll get with that code are mostly permissions related, nothing you can fix unless you check them beforehand and efficiently.